content/enterprise_influxdb/v1.9/administration/manage/security/get-started-auth.md

```diff
@@ -0,0 +1,147 @@
+
+## Available methods for managing authorization
+
+There are three ways to manage authorizations in InfluxDB Enterprise.
+Each is useful in different scenarios.
+They are:
+
+- the [`influx` CLI](#influxql) (1.x) with [InfluxQL](#manage-read-and-write-privileges-with-influxql)
+- [Chronograf](#manage-specific-privileges-with-chronograf)
+- the [InfluxDB Enterprise meta API](#influxdb-enterprise-meta-api)
+
+Each of these allows you to manage permissions for specific users.
+
+### Manage read and write privileges with InfluxQL
+
+{{% note %}}
+InfluxQL provides a quick way to grant read and write privileges.
+However, you can only grant `READ`, `WRITE`, and `ALL PRIVILEGES` privileges with this method.
+For more, see [Manage specific privileges with Chronograf](#manage-specific-privileges-with-chronograf) below.
+{{% /note %}}
+
+InfluxQL can be used to manage basic read and write privileges.
+For example, you can grant Alice the ability to write to a database *X*,
+and then grant Bob the ability to read from that database.
+
+To demonstrate, let's create a new user and allow her some access.
+First, log in as an admin user.
+The simplest way to do this is to use the `auth` command.
+
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.9.5-c1.9.5
+InfluxDB shell version: 1.8.7
+> auth
+```
+
+This will prompt for username and password.
+Enter the credentials for the admin you created above in ["Enable authentication"](#enable-authentication).
+
+Then run:
+
+```sh
+> CREATE USER alice WITH PASSWORD 'admin'
+```
+
+This creates a regular, non-admin user.
+
+Since we are logged in as an admin user, we can create a database.
+Create a database called `test` by running
+
+```sh
+> CREATE DATABASE test
+```
+
+Now we can grant Alice permission to write to the `test` database.
+
+```sh
+> GRANT WRITE ON test TO alice
+```
+
+We can inspect Alice's permissions with
+
+```sh
+> SHOW GRANTS FOR alice
+database privilege
+-------- ---------
+test     WRITE
+```
+
+### Manage specific privileges with Chronograf
+
+The Chronograf user interface can manage the [full set of permissions](/enterprise_influxdb/v1.9/administration/manage/security/authentication_and_authorization-api/#list-of-available-privileges).
+
+The permissions listed in Chronograf (and available through the API) are global for the cluster.
+Outside of [FGA](), the only database-level permissions available are the basic `READ` and `WRITE`.
+These can only be managed using [the `influx` CLI and InfluxQL](#manage-read-and-write-privileges-with-influxql).
+
+To manage privileges in Chronograf:
+
+1. Login to Chronograf as an admin user.
+2. Click on **InfluxDB Admin** → **Users**.
+3. Each user has a dropdown for **Permissions**.
+   Select the desired permissions and click **Apply**.
+
+### InfluxDB Enterprise meta API
+
+For information on using the meta API, see [here](/enterprise_influxdb/v1.9/administration/manage/security/authentication_and_authorization-api).
```

Comment:
Let's omit the extra click for folks by dropping this section and moving link to top